src/app/public/modules/filter/filter-button.component.scss

```diff
@@ -4,6 +4,10 @@
    color: $sky-highlight-color-success;
    border: 2px solid $sky-highlight-color-success;
    padding: 5px 11px; /* offset the thicker border to avoid button size change */
+
+  @include sky-theme-modern {
+    padding: initial;
```

Comment:
In modern theme, it looks like the padding is changing when I click on the button:
![Screen Shot 2021-03-22 at 9 32 03 AM](https://user-images.githubusercontent.com/18354182/111997689-aaaabe00-8af1-11eb-810f-f46a52b1a87a.png)
![Screen Shot 2021-03-22 at 9 32 09 AM](https://user-images.githubusercontent.com/18354182/111997697-ac748180-8af1-11eb-8207-3506fe4c7e08.png)